engine/pmem_allocator/pmem_allocator.cpp

```diff
@@ -115,6 +115,27 @@ PMEMAllocator* PMEMAllocator::NewPMEMAllocator(
     return nullptr;
   }
 
+  uint64_t segment_size = block_size * num_segment_blocks;
+
+  if (pmem_size < segment_size * max_access_threads) {
```

Comment:
please add this to configs check too

Reply:
has already exist.
![image](https://user-images.githubusercontent.com/7602435/162371061-5d67062b-5c4e-46ce-9db0-4d8bf56b78f4.png)